client/package.json

```diff
@@ -7,8 +7,10 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
+    "react-countup": "^6.4.1",
```

Comment:
Try to implement these counting animations without any packages, As asked earlier we have to minimize the use of packages